pkg/ipcache/ipcache.go

```diff
@@ -82,7 +82,8 @@ type IPCache struct {
 
 	listeners []IPIdentityMappingListener
 
-	namedPorts policy.NamedPortMultiMap
+	needNamedPorts bool
```

Comment:
I think it would be helpful context to briefly explain why `needNamePorts` needs to exist / its role.

Reply:
+1 I'm also lost.

Reply:
Adding this comment:
```
	// needNamedPorts is initially 'false', but will be changd to 'true' when the
	// clusterwide named port mappings are needed for network policy computation
	// for the first time. This avoids the overhead of maintaining 'namedPorts' map
	// when it is known not to be needed.
	// Both needNamedPorts and namedPorts are protected by 'mutex'.
```

Reply:
Thanks for adding that. Sounds like an optimization. Just out of curiosity, did we measure the impact of this?